src/Sulu/Component/SmartContent/ContentType.php

```diff
@@ -245,10 +245,10 @@ public function getContentData(PropertyInterface $property)
         $filters['excluded'] = [$property->getStructure()->getUuid()];
 
         // default value of tags/category is an empty array
-        if (!array_key_exists('tags', $filters)) {
+        if (!array_key_exists('tags', $filters) || null === $filters['tags']) {
```

Comment:
@wachterjohannes why not use `!isset($filters['tags'])`  php.net/manual/en/function.isset.php